src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1314,6 +1314,15 @@
   <data name="PreferTypedStringBuilderAppendOverloadsRemoveToString" xml:space="preserve">
     <value>Remove the ToString call</value>
   </data>
+  <data name="PreferStringContainsOverIndexOfDescription" xml:space="preserve">
+    <value>Calls to 'String.IndexOf' where the result is then just compared to -1 can instead be replaced by 'String.Contains'</value>
```

Comment:
The wording here confused me the first couple of times I read it. Maybe it can be simplified?:
```suggestion
    <value>Calls to 'String.IndexOf' where the result is compared to -1 can be replaced by 'String.Contains'</value>
```